contrib/filter-repo-demos/git_lint_history.py

```diff
@@ -142,42 +151,47 @@ def lint_non_binary_blobs(blob, metadata):
     filename = '.git/info/tmpfile'
     with open(filename, "wb") as f:
       f.write(blob.data)
-    subprocess.check_call(lint_args.command + [filename])
+    subprocess.check_call(shlex.split(lint_args.command[0]) + [filename])
     with open(filename, "rb") as f:
       blob.data = f.read()
     os.remove(filename)
 
 def main(argv1=None):
-  global is_relevant, lint_args, tmpdir, cat_file_process, filter
+  global is_relevant, lint_args, tmpdir, cat_file_process, rfilter
 
   if isinstance(argv1, str):
     argv1 = shlex.split(argv1)
 
-  lint_args = parser.parse_args(argv1)
+  lint_args, rest = parser.parse_known_args(argv1)
   if not lint_args.command:
     raise SystemExit("Error: Need to specify a lint command")
-
+  if lint_args.relevant_glob:
+    if lint_args.relevant:
+      raise SystemExit("--relevant and --relevant-glob given both")
```

Comment:
This error message seems hard to understand.  Perhaps: "Error: --relevant and --relevant-glob are incompatible"?